lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -919,13 +919,13 @@ export class DocumentListComponent implements OnInit, OnChanges, OnDestroy, Afte
         this.error.emit(err);
     }
 
-    getPreselectNodesBasedOnSelectionMode(): NodeEntry[] {
-        return this.hasPreselectNodes() ? (this.isSingleSelectionMode() ? [this.preselectNodes[0]] : this.preselectNodes) : [];
+    getPreselectedNodesBasedOnSelectionMode(): NodeEntry[] {
+        return this.hasPreselectedNodes() ? (this.isSingleSelectionMode() ? [this.preselectNodes[0]] : this.preselectNodes) : [];
     }
 
     onPreselectNodes() {
-        if (this.hasPreselectNodes()) {
-            const preselectedNodes = [...this.isSingleSelectionMode() ? [this.data.getPreselectRows()[0]] : this.data.getPreselectRows()];
+        if (this.data.hasPreselectedRows()) {
```

Comment:
this is changing the behaviour...compared to the previous code, are we sure ?

Reply:
To be honest I believe this is the right thing to do because in the next lines we access the `this.data.getPreselectedRows()[0] or this.data.getPreselectedRows()` based on if the selection is single or multiple.
So to me seems correct we should check the if(this.data.hasPreselectedRows) as this is the one we are trying to access in the next line. I don't know what the author of the code wanted to achieve with this but I believe this is the correct thing.